docs/source/howto/usage/kwargs.rst

```diff
@@ -108,7 +108,8 @@ brain_mask_definition: instance from `AFQ.definitions.image`, optional
 bundle_info: strings, dict, or BundleDict, optional
 	List of bundle names to include in segmentation,
 	or a bundle dictionary (see BundleDict for inspiration),
-	or a BundleDict. See `Defining Custom Bundle Dictionaries`
+	or a BundleDict. See `Defining Custom Bundle Dictionaries` 
+	(https://yeatmanlab.github.io/pyAFQ/reference/bundledict.html)
```

Comment:
This would create a text hyperlink (see [here](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#hyperlinks))
```suggestion
	or a BundleDict. See `Defining Custom Bundle Dictionaries <https://yeatmanlab.github.io/pyAFQ/reference/bundledict.html>`_
```